### ISSUE TITLE
view: when refreshing the page the data is loaded twice

### ISSUE THREAD

**Issue by teosarca** [metasfresh/metasfresh-webui-frontend#1063]
### Type of issue
Bug

### Steps to reproduce
* open a view, any view, e.g. https://w101.metasfresh.com:8443/window/540189
* press F5 (refresh)

Remark that the endpoint used to fetch data is called twice:
![image](https://user-images.githubusercontent.com/1244701/28462959-8585a6b2-6e26-11e7-90b0-d180dae34fd8.png)

### Expected behavior
Data shall be fetched from backend only once.

**Comment by cadavre** [comment]
Probably a duplicate of #1048

**Comment by metas-lc** [comment]
IT
checked several windows, including 540189 => data is fetched only once
